src/cart/crt_rpc.c

```diff
@@ -441,8 +441,7 @@ crt_register_proto_fi(crt_endpoint_t *ep)
 	if (rc != 0)
 		return -DER_MISC;
 
-	rc = crt_proto_query(ep, cpf.cpf_base, &cpf.cpf_ver,
-			     1, crt_pfi_cb, &pfi);
+	rc = crt_proto_query(ep, cpf.cpf_base, &cpf.cpf_ver, 1, 60, crt_pfi_cb, &pfi);
```

Comment:
would be better to pass 0 here and in cart we just not set the timeout of the value passed is 0.